packages/contracts/test/contracts/OVM/chain/OVM_CanonicalTransactionChain.spec.ts

```diff
@@ -556,6 +556,67 @@ describe('OVM_CanonicalTransactionChain', () => {
       ).to.equal(true)
     })
 
+    it('should reject a valid queue transaction if QueueOrigin is modified to SEQUENCER_QUEUE', async () => {
+      const entrypoint = NON_ZERO_ADDRESS
+      const gasLimit = 500_000
+      const data = '0x' + '12'.repeat(1234)
+
+      const timestamp = (await getEthTime(ethers.provider)) + 100
+      await setEthTime(ethers.provider, timestamp)
+      await OVM_CanonicalTransactionChain.enqueue(entrypoint, gasLimit, data)
+
+      const blockNumber = await ethers.provider.getBlockNumber()
+
+      await appendSequencerBatch(
+        OVM_CanonicalTransactionChain.connect(sequencer),
```

Comment:
Style nit: placing brackets/parens on their own lines is verbose and makes it more difficult to read. This could be condensed and remove multiple lines like so:

```js
        OVM_CanonicalTransactionChain.connect(sequencer), {
          shouldStartAtElement: 0,
          totalElementsToAppend: 1,
          contexts: [{
              numSequencedTransactions: 0,
              numSubsequentQueueTransactions: 1,
              timestamp,
              blockNumber,
          }],
          transactions: [],
        })
```

Reply:
I agree, that's much nicer,  but this is currently enforced by the linter, and appears 20 or so other times in the same file. 
Also worth noting that this branch still uses tslint, whereas `develop` is now on eslint, so it may change when this is rebased.

Reply:
Yeah, the [array-bracket-newline](https://github.com/ethereum-optimism/optimism/blob/develop/.eslintrc.js) rule is not in the new [eslint config](https://github.com/ethereum-optimism/optimism/blob/develop/.eslintrc.js). This would be nice to fix, as there are much worse cases of this style: 

https://github.com/ethereum-optimism/optimism/blob/e6e85a6220de0c4e1d28749bc4b1d27d4dd7e4c1/packages/contracts/test/contracts/OVM/chain/OVM_CanonicalTransactionChain.spec.ts#L968-L970

Reply:
Ok, thanks for noting which rule it is, perhaps we can do a big re-lint in the future